rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -54,7 +54,127 @@ paths:
           $ref: '#/components/responses/NotImplemented'
         default:
           $ref: '#/components/responses/DefaultError'
-
+  /transform/{TransformId}:
+    parameters:
+      - $ref: '#/components/parameters/TransformId'
+    get:
+      summary: Poll transformation request status
+      description: >
+        After posting a transformation request, the client can use the returned URI to poll for the transform's completion at this endpoint.
+      operationId: poll
+      tags:
+        - transform
+      responses:
+        '200':
+          description: The transform is still in progress.
+          headers:
+            Content-Version:
+              $ref: '#/components/headers/ContentVersion'
+        '201':
+          description: Transform complete. Returning metadata locations.
+          headers:
+            Content-Version:
+              $ref: '#/components/headers/ContentVersion'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    format:
```

Comment:
```suggestion
                    type:
```

Reply:
Renamed to `metadataType` (I wanted to avoid just type since that's a "keyword" for these JSON spec being used here).